ng/.angular-cli.json

```diff
@@ -17,7 +17,6 @@
       "test": "test.ts",
       "tsconfig": "tsconfig.app.json",
       "testTsconfig": "tsconfig.spec.json",
-      "prefix": "app",
```

Comment:
🤷‍♀️ This was causing tslint to bug me about prefixing my selectors w/ `app-`. I think because of the global namespace that selectors live in?